addon/services/alexandria-documents.js

```diff
@@ -13,7 +13,7 @@ import { ErrorHandler } from "ember-alexandria/utils/error-handler";
  */
 const fileHasValidMimeType = (file, category) =>
   category.allowedMimeTypes?.includes(
-    file.type || mime.getType(file.name ?? file.title),
+    file.mimeType || mime.getType(file.name ?? file.title),
```

Comment:
```suggestion
    file.mimeType
```

Let's drop the fallback - every file has a mime type.

Reply:
I tried it, but then there are failing tests because of the upload itself. 
This code will be executed when you upload a file (or multiple files):
```
await this.documents.upload(
      this.args.filters.categories,
      event.dataTransfer.files,
    );
```
So the evaluation of the mimeType is completely in the frontend, which means, that the mimeType attribute in the file isn't set yet.

What would you prefer:
- Option 1: add another "fileHasValidMimeType" check, where only the mimeType is checked when files are moved (e.g. "hasValidMimeTypeByFileExtension" and "hasValidMimeType")
- Option 2: keep the function like it is to handle both checks (upload: frontend side mimeType evaluation and moving: use evaluation of mimeType, which was set from the backend)

Reply:
Option 3: Check if the passed models `isNew` property is `true` - if yes, check the filename (not the title) if no, check the mime type.